apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.OrderingKeyTester/Program.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Grpc.Core;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Google.Cloud.PubSub.V1.OrderingKeyTester
+{
+    class Program
+    {
+        private class InputLine
+        {
+            public InputLine(string line)
+            {
+                var parts = line.Split(',');
+                if (parts.Length != 2)
+                {
+                    throw new Exception("Expected two parts in input file.");
+                }
+                OrderingKey = parts[0].Trim('"');
+                Message = parts[1].Trim('"');
+            }
+
+            public string OrderingKey { get; }
+            public string Message { get; }
+        }
+
+        static int Main(string[] args)
```

Comment:
I think it would be good to break this into a bunch of methods, then make Main just call the methods in turn.